drone/oc-integration-tests/start.sh

```diff
@@ -0,0 +1,11 @@
+REVAD=~/www/reva/cmd/revad/revad
```

Comment:
This does not seem to be used. The commands are inline in the drone pipeline(s).

Reply:
just a local file that is not needed in CI